src/Client.Infrastructure/ApiClient/FSHApi.cs

```diff
@@ -4626,23 +4630,9 @@ public virtual async System.Threading.Tasks.Task<System.Collections.Generic.ICol
                             return objectResponse_.Object;
                         }
                         else
-                        if (status_ == 400)
```

Comment:
Are these generated by NSWAG? or you had to manually change the code?

Reply:
In this file was all NSWAG. Nothing manually